Taskfile.dist.yaml

```diff
@@ -2,6 +2,27 @@ version: "3"
 
 silent: true
 
+vars:
+  os: "{{OS}}"
+  arch: '{{ternary "arm64" ARCH (eq ARCH "arm")}}'
+  mode: '{{ternary "cgo" "go" (eq .os "linux")}}'
+  package: "github.com/werf/trdl/client/cmd/trdl"
```

Comment:
I don't get it. There are multiple packages in the repo. I don't think we need this variable, unless you call it "module_client_path" or something. But then also add "module_server_path" and others.